ddev/src/ddev/cli/ci/codeowners.py

```diff
@@ -0,0 +1,70 @@
+import click
+from datadog_checks.dev.tooling.codeowners import CodeOwners
+
+from ddev.cli.application import Application
+
+
+@click.command()
+@click.option('--pr', help='Check the codeowners for the given PR')
+@click.option('--sha', help='Check the codeowners for the given sha')
+@click.option('--files', help='Check the codeowners for the given files separated by commas')
+@click.option('--per-file', is_flag=True, help='Output the codeowners for each file')
+@click.pass_obj
+def codeowners(app: Application, pr: str, sha: str, files: str, per_file: bool):
+    """
+    Check the codeowners for the given PR, commit, or files.
+    """
+    codeowners_file = app.repo.path / '.github' / 'CODEOWNERS'
+    with open(codeowners_file, 'r') as f:
+        codeowners_content = f.readlines()
+    codeowners: CodeOwners = CodeOwners("\n".join(codeowners_content))
```

Comment:
```suggestion
	codeowners_file = app.repo.path / ".github" / "CODEOWNERS"
	
	if not codeowners_file.exists():
	    raise FileNotFoundError(f"{codeowners_file} does not exist")
	
	codeowners_content = codeowners_file.read_text()
	codeowners = CodeOwners(codeowners_content)
```
suggestion:
- Check for codeowners file existence and raise an exception if it doesn't exist.
- Remove unnecessary type hint in the `codeowners` variable, the CodeOwner type/class is already shown in the value.
- Using the [Path.read_text](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text) method opens the file, reads it in text-mode and returns its contents making sure it gets closed, using this method is simpler and removes the need for manually using a context manager `with`.